docs/APIReference-Editor.md

```diff
@@ -163,6 +163,12 @@ convert that `ContentBlock` into an `unordered-list-item`.
 At Facebook, we also use this to convert typed ASCII quotes into "smart" quotes,
 and to convert typed emoticons into images.
 
+#### handlePastedText
+```
+handlePastedText?: (text: string, html: string) => boolean
```

Comment:
`html` should be an optional parameter. It may be a plaintext paste with no HTML available.

```javascript
handlePastedText?: (text: string, html?: string) => boolean
```